languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -12456,46 +12456,51 @@ though, and has since been slightly modified)-->
         <example>Parce qu'elles sont mises à part, celles-là doivent avoir été choisies.</example>
       </rule>
     </rulegroup>
-    <rule id="AUX_ETRE_VCONJ" name="Auxiliaire suivi d’un verbe conjugué">
-      <antipattern>
-        <token postag_regexp="yes" postag="V.*"/>
-        <token postag="R.*" postag_regexp="yes" spacebefore="no"/>
-      </antipattern>
-      <pattern>
-        <token postag="V etre .*" postag_regexp="yes">
-          <exception regexp="yes" scope="previous">.+'</exception>
-          <exception inflected="yes" scope="previous">avoir</exception>
-          <exception>soit</exception>
-          <exception scope="previous">ce</exception>
-        </token>
-        <marker>
-          <token postag="V (inf|ind|sub|con|ppr).*" postag_regexp="yes">
-            <exception postag="[ZN] .*" postag_regexp="yes"/>
-          </token>
-        </marker>
-      </pattern>
-      <message>Après l’auxiliaire « \1 », le verbe devrait être au participe passé. Quant à l’accord, celui-ci s’accorde avec le COD s’il est placé avant l’auxiliaire ; sinon il restera au masculin singulier.</message>
-      <suggestion><match no="3" postag="V.*" postag_regexp="yes" postag_replace="V ppa m s"/></suggestion>
-      <suggestion><match no="3" postag="V.*" postag_regexp="yes" postag_replace="V ppa f s"/></suggestion>
-      <suggestion><match no="3" postag="V.*" postag_regexp="yes" postag_replace="V ppa m p"/></suggestion>
-      <suggestion><match no="3" postag="V.*" postag_regexp="yes" postag_replace="V ppa f p"/></suggestion>
-      <url>http://grammaire.reverso.net/4_1_05_accord_du_participe_passe_avec_avoir.shtml</url>
-      <example correction="parti|partie|partis|parties">Il est <marker>partit</marker> tôt.</example>
-      <example>Il est arrivé</example>
-      <example>Il est parti tôt.</example>
-      <example correction="arrivé|arrivée|arrivés|arrivées">Il est <marker>arriver</marker></example>
-      <example>L’<marker>été</marker> est chaud.</example>
-      <example>L’objectif est de réussir</example>
-      <example>Partir, c’est mourir un peu.</example>
-      <example>Il faut être <marker>aveugle</marker> pour ne pas voir ça.</example>
-      <example>Il était <marker>officier</marker>.</example>
-      <example>Crois-tu que je sois aveugle ?</example>
-      <example>Les Allemandes sont toutes des délurées.</example>
-      <example>Tu as été prendre du bois.</example>
-      <example>La fille là-bas est Marie.</example>
-      <example>Ce serait attaquer les droits de l’homme.</example>
-      <example>Il faut soit partir, soit attendre.</example>
-    </rule>
+      <rule id="AUX_ETRE_VCONJ" name="Auxiliaire suivi d’un verbe conjugué">
```

Comment:
We will add `default="temp_off"` here (and to the other rules). With this we check that there are not too many false alarms on a corpus before enabling the rule for the users.